Makefile

```diff
@@ -1,6 +1,8 @@
 # url to a running release-manager
 URL=http://localhost:8080
 FILE=payload.json
+FILE2=changeBasePayload.json
+FILE3=editedButNotBranch.json
```

Comment:
These names could be a bit more descriptive. :) Maybe related to the make targets they support?

Reply:
Yes sir https://github.com/lunarway/release-manager-bot/pull/22/commits/75a58cf490b22a63b74fe04fcc444d86362efadf